packages/react/src/combobox/test-utils.ts

```diff
@@ -197,3 +197,14 @@ export const COUNTRY_OPTIONS = [
 	'Zambia',
 	'Zimbabwe',
 ].map((country) => ({ label: country, value: country }));
+
+export const STATE_OPTIONS = [
```

Comment:
This was duplicated across multiple test files, so it's usage has been consolidated and put here.